compiler/global/scoping.ml

```diff
@@ -606,6 +606,7 @@ let block locals body env_pat env
 (** Scoping an expression *)
 let rec expression env { desc = desc; loc = loc } =
   let desc = match desc with
+    | Edummy -> Zelus.Edummy
```

Comment:
Have you tried using Evoid keyword that is already defined in "zparsetree.ml". If this works, it will avoid code duplication.